app/models/identity.rb

```diff
@@ -2,6 +2,10 @@ class Identity < ActiveRecord::Base
   include NonNullUuid
 
   belongs_to :user
+  belongs_to :sp,
```

Comment:
That makes sense. 

It is a tad confusing to have this defined *just* for the ERD diagram (we do not use `identity.sp` anywhere) but perhaps our only option here?